src/styles/components/pages/tutorials/_Tutorials.scss

```
@@ -0,0 +1,73 @@
+.tutorials {
+  background-color: $background-tuto-purple;
+  background-image: url('/onboarding-1@3x.jpg');
+  background-position-x: center;
+  background-position-y: bottom;
+  background-repeat: no-repeat;
+  background-size: contain;
+  color: $white;
+  font-size: rem(18px);
+  height: 100%;
+  line-height: 1.25;
+  text-align: center;
+  width: 100%;
+
+  .calendar {
+    margin-top: 9vh;
+  }
+
+  .first-tutorial-text {
+    margin: 5.5vh 12vw 0;
+
+    .text-highlight {
+      color: $purple-blue;
+      font-weight: 500;
+      margin: 0 2px;
+      padding: 1px 3px;
+      position: relative;
+      z-index: 10;
+
+      &::after {
+        background-color: $white;
+        border-radius: 2px;
+        content: '';
+        height: 100%;
+        left:0;
+        position: absolute;
+        width: 100%;
+        z-index: -1;
+      }
+
+      &.text-2-ans::after {
+        transform: rotate(1deg);
+      }
+
+      &.text-500-euros::after {
+        transform: rotate(-1deg);
+      }
+    }
+  }
+
+  .slider-points {
+    bottom: rem(32px);
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    left: 50%;
+    position: absolute;
+    transform: translate(-50%, 0);
+
+    .slider-point {
+      background-color: $white;
+      border-radius: 8px;
+      height: 8px;
+      margin: 0 5px;
+      opacity: .5;
+      width: 8px;
```

```suggestion
      width: rem(8px);
```
Par défaut, là où il y a du pixel, faut s'arranger pour le transformer en rem pour améliorer l'accessibilité (zoom).